src/components/AudioResultsList.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <section>
+    <div v-show="!isFetchingAudios && includeAnalytics" class="results-meta">
+      <span class="caption font-semibold">
+        {{ audiosCount }}
+      </span>
+    </div>
+    <div class="px-6">
+      <AudioTrack
+        v-for="audio in audios"
+        :key="audio.id"
+        :audio="audio"
+        :is-compact="true"
+      />
+    </div>
+    <div class="load-more">
+      <button
+        v-show="!isFetchingAudios && includeAnalytics"
+        class="button"
+        :disabled="isFinished"
+        @click="onLoadMoreAudios"
+        @keyup.enter="onLoadMoreAudios"
+      >
+        <span v-if="isFinished">{{
+          $t('browse-page.no-more', {
+            type: $t('browse-page.search-form.audio'),
+          })
+        }}</span>
+        <span v-else>{{ $t('browse-page.load') }}</span>
+      </button>
+      <LoadingIcon v-show="isFetchingAudios" />
+    </div>
```

Comment:
Do we need to handle fetching errors here, just like in the images page?

Reply:
Right! Adding it in a moment :)

Reply:
Done in [`493f38d`](https://github.com/WordPress/openverse-frontend/pull/200/commits/493f38d5df9f5c71d7d0007142f2a901f1308c65).